bot/src/module/song.rs

```diff
@@ -258,10 +258,12 @@ impl command::Handler for Handler {
                     "" => None,
                     other => Some(other.to_string()),
                 });
+                ctx.user.as_owned_user().respond("Closed player from further requests.");
```

Comment:
`ctx.respond(...)` should be a shorthand which doesn't require taking ownership for this.